InnerEye/ML/run_ml.py

```diff
@@ -254,6 +256,20 @@ def innereye_config(self) -> DeepLearningConfig:
                              f"configuration is of type {type(self.model_config)}")
         return self.model_config
 
+    @property
+    def config_namespace(self) -> str:
+        """
+        Returns the namespace of the model configuration object.
```

Comment:
can you maybe add examples in docu? and explain what is mean by "namespace" to start with? for example, for the Lung model, what is the namespace that is returned?